modules/local/chunkfasta.nf

```diff
@@ -2,9 +2,7 @@ process CHUNKFASTA {
     tag "${meta.id}"
     label "process_medium"
 
-    if (params.enable_conda) {
-        exit 1, "Conda environments cannot be used when using the chunkfasta process. Please use docker or singularity containers."
-    }
+    conda (params.enable_conda ? "conda-forge::pyfasta=0.5.2-1" : null)
     container "quay.io/biocontainers/pyfasta:0.5.2--py_1"
```

Comment:
Can you please update with singularity and docker as it is with standard modules?